releasenotes/README.md

```diff
@@ -16,6 +16,10 @@ If you use [autocomplete](https://developer.salesforce.com/docs/atlas.en-us.sfdx
 
 These changes are in the Salesforce CLI release candidate. We plan to include these changes in next week's official release. This list isn't final and is subject to change.
 
+* NEW: Some commands (`force:org:open`, `force:org:display`, `force:user:display`, `force:source:open`) output tokens and URLs with tokens embedded in them.  They now include a warning about the risks of sharing that sensitive information. 
```

Comment:
Seems like a verb is missing. Some commands (yadda) **use** output tokens...

Reply:
`output` is a verb but can also be a noun.  Would you rather say 'some commands **print** tokens?'

Reply:
Doh! Words are hard. I misread it - so others might. what about "display" ? If not, we can stick with output.